npm/private/npm_package_store.bzl

```diff
@@ -158,6 +158,8 @@ def _impl(ctx):
     transitive_files = []
     direct_ref_deps = {}
 
+    npm_package_store_deps = ctx.attr.src[NpmPackageInfo].npm_package_store_deps.to_list() if ctx.attr.src else []
```

Comment:
we can avoid the to_list here by keeping these as depsets. You're just making them transitives of a later depset it seems

Reply:
This one we're iterating on below so we need to convert. This code path is only for manually linked 1st party deps; it is an empty depset in the happy path pnpm-lock file path